R/stage_nwis_ts.R

```diff
@@ -113,13 +117,18 @@ stage_nwis_ts <- function(sites, var, times, folder = tempdir(), verbose = FALSE
     for (i in 1:length(un_sites)){
       
       site <- parse_site_name(un_sites[i])
-      site_data <- filter(nwis_data, site_no == site) %>%
-        select_('DateTime', var) %>%
+      site_data <- filter(nwis_data, site_no == site)
+      which.var <- select(site_data, -DateTime, -site_no) %>% 
```

Comment:
wow, what a pain! is it really selecting `c(1)[1]`, or does the first chunk of code get you to a rowless df (more like `c()[1]`?